scripts/codegen-firecracker.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Test for platform
+platform=`uname -sp`
+if [[ $platform != "Linux x86_64" ]]; then
+  echo "Codegen script only works on Linux x86 platform"
+  exit 0
+fi
+
+# At the moment, we only test codegen for the virtio module
+cd /tmp
+git clone https://github.com/firecracker-microvm/firecracker.git
```

Comment:
We might want to pin this to a particular commit?

Reply:
Could this be a submodule instead? That's a standard way to pin a specific commit, plus tooling support like Dependabot. :)